setup.cfg

```diff
@@ -6,7 +6,8 @@ exclude =
     __pycache__,
     build,
     debian,
-    script
+    script,
+    venv
```

Comment:
This change seems unrelated? Consider cherry-picking to `master`?

Reply:
It seemed related, but mostly during a merge fix-up. Without this `flake8` tried to lint the venv, which naturally took ages.